docs/content/integrations/airbyte.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: "Using Dagster with Airbyte"
+description: Integrate your Airbyte connections into Dagster.
+---
+
+# Using Airbyte with Dagster
+
+Dagster can orchestrate your Airbyte connections, making it easy to chain an Airbyte sync with upstream or downstream steps in your workflow.
+
+This guide focuses on how to work with Airbyte connections using Dagster's [software-defined asset (SDA)](/concepts/assets/software-defined-assets) framework.
+
+---
+
+## Airbyte connections and Dagster software-defined assets
+
+An [Airbyte connection](https://docs.airbyte.com/understanding-airbyte/connections/) defines a series of data streams which are synced between a source and a destination. During a sync, each stream is replicated in the destination, typically as one or more tables. Dagster represents each of the replicas generated in the destination as a software-defined asset. This enables you to easily:
+
+- Visualize the streams involved in an Airbyte connection and execute a sync from Dagster
+- Define downstream computations which depend on replicas produced by Airbyte
+- Track historical metadata and logs for each data stream
+- Track data lineage through Airbyte and other tools
+
+---
+
+## Loading Airbyte assets into Dagster
```

Comment:
If I understand correctly, this omits the `build_airbyte_assets` way. Unless we're confident that it should be deprecated, I think it's important to mention it. Otherwise, the people who rely on it will be confused.

Reply:
Yeah, I've gone ahead and added another section for it. Maybe we should just mention it and not have a full tutorial though if we want to de-emphasize this approach - what do you think?

Reply:
I think this looks good. The build_airbyte_assets section is short and sweet.